src/pydata_sphinx_theme/assets/styles/components/_search.scss

```diff
@@ -91,30 +98,24 @@
     margin-top: 0.5rem;
     width: 90%;
     max-width: 800px;
-  }
+    background-color: transparent;
+    padding: $focus-ring-width;
+    border: none;
```

Comment:
Had to reset padding and border. Otherwise browsers provide default values for padding and border on dialog boxes.